tests/grain_size_strain_pinned_state.prm

```diff
@@ -0,0 +1,52 @@
+# A test for the grain size material model using pinned state damage formulation. 
+# Over time the model maintains nearly constant temperatures, pressures and
+# strain rates. The grain size in the test approaches the equilibrium
+# grain size for t -> infinity. At 200 Myr the equilibrium grain size should
+# be 0.0165746 m.
```

Comment:
Can you add a comment here that because we can not let the test run for such a long time, the test result is very different from that number?
And it would also be good to refer to the python notebook that computes the number.